flink-connector-aws/flink-connector-dynamodb/src/test/java/org/apache/flink/connector/dynamodb/table/RowDataElementConverterTest.java

```diff
@@ -45,10 +48,17 @@ public class RowDataElementConverterTest {
                     DataTypes.FIELD("partition_key", DataTypes.STRING()),
                     DataTypes.FIELD("payload", DataTypes.STRING()));
     private static final RowDataElementConverter elementConverter =
-            new RowDataElementConverter(DATA_TYPE);
+            new RowDataElementConverter(DATA_TYPE, null);
     private static final SinkWriter.Context context = new UnusedSinkWriterContext();
     private static final RowDataToAttributeValueConverter rowDataToAttributeValueConverter =
-            new RowDataToAttributeValueConverter(DATA_TYPE);
+            new RowDataToAttributeValueConverter(DATA_TYPE, null);
+
+    private static final Set<String> primaryKeys =
+            new HashSet<>(Collections.singletonList("partition_key"));
```

Comment:
nit: You can use `Collections.singleton(...)` to create Set (immutable) with a single element.

Reply:
Implemented these changes now.